src/tribol/interface/tribol.cpp

```diff
@@ -921,6 +1110,96 @@ integer update( integer cycle, real t, real &dt )
       }
 
       CouplingScheme* couplingScheme  = csManager.getCoupling(csIndex);
+
+      // update redecomp meshes if supplied mfem data
+      if (couplingScheme->hasMfemData())
+      {
+         auto mfem_data = couplingScheme->getMfemMeshData();
+         axom::Array<int> mesh_ids {2, 2};
+         mesh_ids[0] = mfem_data->GetMesh1ID();
```

Comment:
Minor, but in places like the coupling scheme, it has always been 'meshId#' not 'mesh#Id'. At this point I'm programmed to type the former. Can we refactor so we are consistent?

Reply:
I'm open to changing, but we will have to think about this in the context of the other methods in this class.  Let's think about what makes sense overall.